### ISSUE TITLE
Flow type definitions

### ISSUE THREAD

**Issue by nitingupta910** [reactstrap/reactstrap#686]
Addition of reactstrap flow type definitions to [flow-typed](https://github.com/flowtype/flow-typed/tree/master/definitions/npm) would be really useful.

**Comment by TheSharpieOne** [comment]
See discussion in #276

**Comment by nitingupta910** [comment]
Yes, I saw that and I opened this issue to keep the focus on flow types.

**Comment by Falco101** [comment]
I would love to see this, reactstrap has become a must include tool for rapid UI development and I've started integrating flow into my projects, having better type checking and intellisense would be fantastic!
